### ISSUE TITLE
Failed prop type: Invalid prop `inputContainerStyle` supplied to `Input`.

### ISSUE THREAD

**Issue by freekingg** [necolas/react-native-web#1889]
```
Warning: Failed prop type: Invalid prop `inputContainerStyle` supplied to `Input`.
```
```
Warning: Invalid DOM property `enterkeyhint`. Did you mean `enterKeyHint`?
```
```
index.js:1 Warning: Failed prop type: Invalid prop `buttonStyle` supplied to `Button`.
Warning: Failed prop type: Invalid prop `containerStyle` supplied to `Button`.
```
my version
```
"react-native-web": "0.14.0",
"react-native-elements": "^3.1.0",
"react-native": "0.63.4",
```
mycode
```
<Input
            ref={pwdInputEl}
            placeholder="登录密码"
            placeholderTextColor="rgba(255,255,255,0.60)"
            leftIcon={{
              type: 'font-awesome',
              name: 'lock',
              color: 'rgba(255,255,255,0.60)',
            }}
            inputContainerStyle={styles.inputContainerStyle}
            inputStyle={{
              color: global.colors.base,
            }}
            errorMessage={pwdErrorMsg}
            onChangeText={text => {
              setPwdErrorMsg('');
              setFormData({
                ...formData,
                pwd: text,
              });
            }}
          />
```

I combine React - Native Elements what should I do

**Comment by necolas** [comment]
Caused by RN Elements forwarding props that are not DOM props - https://github.com/react-native-elements/react-native-elements/blob/next/src/input/Input.js#L133

**Comment by curiousdustin** [comment]
@necolas , if I comment out that line, I still get this error.

``` js
            // {...patchWebProps(attributes)}
```

**Comment by curiousdustin** [comment]
For anyone else running into this, the issue for me was the type of the value supplied to `containerStyle`. 

Instead of a simple object, I was supplying a style object created using 
``` js 
StyleSheet.create({ myContainerStyle: {} })
```

Apparently the `PropTypes` checking within React Native Elements does not like this.

**Comment by hungvu193** [comment]
Well, this issue wont crash the app, but It look so uncomfortable, is there any way to fix this?

**Comment by Montenedro** [comment]
> For anyone else running into this, the issue for me was the type of the value supplied to `containerStyle`.
> 
> Instead of a simple object, I was supplying a style object created using
> 
> ```js
> StyleSheet.create({ myContainerStyle: { /*...*/ } })
> ```
> 
> Apparently the `PropTypes` checking within React Native Elements does not like this.

effectively this solution worked for me. Don't use React-native StyleSheet to create styles, react-native-elements doesn't handle them very well

**Comment by hexadecy** [comment]
Workaround, pass the style in an array:
```jsx
<Input
  inputStyle={[style]}
  inputContainerStyle={[styles.inputContainerStyle]}
  containerStyle={[styles.containerStyle]}
/>
```

**Comment by hexadecy** [comment]
It's fixed in React-Native-Elements 3.3.0+
